Automattic/wp-calypso
# The state and step scaffolding for the website-content step in DIFM onboarding

#### Changes proposed in this Pull Request

This change adds the website-content step scaffolding and the required state infrastructure!
![image](https://user-images.githubusercontent.com/3422709/150344826-e8e2f61e-0206-4f7f-b951-1c10a659a973.png)


#### Testing instructions
* Make sure above is visible `/start/site-content-collection`
* Run the unit tests to make sure nothing fails
